m4/ax_check_libsemigroup.m4

```diff
@@ -15,7 +15,8 @@ AC_DEFUN([AX_CHECK_LIBSEMIGROUPS], [
                           [libsemigroups >= $REQUI_LIBSEMIGROUPS_VERSION],
 			  [need_included_libsemigroups=no],
 			  [need_included_libsemigroups=yes])],
-	[AC_MSG_NOTICE([ignoring flag --with-external-libsemigroups, the Semigroups configure file was created on a system without m4 macros for pkg-config available...])])
+	[AC_MSG_NOTICE([ignoring flag --with-external-libsemigroups, the Semigroups configure file
+                        was created on a system without m4 macros for pkg-config available...])])
```

Comment:
I think this should be undone, it causes the error to change from
```
...
checking for awk... awk
configure: using included libsemigroups...
configure: error: libsemigroups is required, clone or download the repo from https://github.com/libsemigroups/libsemigroups into this directory
```
which is sensibly wrapped in my terminal, to
```
checking for awk... awk
configure: using included libsemigroups...
configure: error: libsemigroups is required, clone or download the repo from
                                  https://github.com/libsemigroups/libsemigroups into this directory
```
which is rather weird.

Reply:
Agreed, I think this change should be undone.

Reply:
There are no two equal terminals, besides, such long lines of code aren't helping.
I can try to make sure the output is formatted better.

Reply:
Have you tested the message? It was perfectly legible for me before your changes but not anymore afterwards. Was the experience different for you?

Reply:
I changed it to avoid extra-long lines in the source, most of all.